qa/src/test/java/io/camunda/migrator/qa/history/HistoryDecisionMigrationTest.java

```diff
@@ -51,38 +72,123 @@ public void migrateHistoricDecisionWithRequirements() {
+    });
+  }
+
+  @Test
+  public void migrateHistoricDecisionInstance() {
+    // given
+    Date now = ClockUtil.now();
+    ClockUtil.setCurrentTime(now);
+    deployer.deployCamunda7Decision("simpleDmn.dmn");
+    deployer.deployCamunda7Process("businessRuleProcess.bpmn");
+    Map<String, Object> variables = Variables.createVariables().putValue("inputA", stringValue("A"));
+    runtimeService.startProcessInstanceByKey("businessRuleProcessId", variables);
+
+    // when
+    historyMigrator.migrate();
+
+    // then
+    List<ProcessInstanceEntity> migratedProcessInstances = searchHistoricProcessInstances("businessRuleProcessId");
+    assertThat(migratedProcessInstances).singleElement();
+    List<DecisionDefinitionEntity> migratedDecisions = searchHistoricDecisionDefinitions("simpleDecisionId");
+    assertThat(migratedDecisions).singleElement();
+    List<FlowNodeInstanceEntity> migratedFlowNodeInstances =
+        searchHistoricFlowNodesForType(migratedProcessInstances.getFirst().processInstanceKey(), BUSINESS_RULE_TASK);
+    assertThat(migratedFlowNodeInstances).singleElement();
+    List<DecisionInstanceEntity> migratedInstances = searchHistoricDecisionInstances("simpleDecisionId");
+    HistoricDecisionInstance legacyInstance = c7Client.getHistoricDecisionInstanceByDefinitionKey("simpleDecisionId");
+
+    assertThat(migratedInstances).singleElement().satisfies(instance -> {
+      assertThat(instance.decisionInstanceId()).isEqualTo(instance.decisionInstanceKey() + "-" + legacyInstance.getId());
+      assertThat(instance.decisionInstanceKey()).isNotNull();
+      assertThat(instance.state()).isEqualTo(DecisionInstanceEntity.DecisionInstanceState.UNSPECIFIED);
+      assertThat(instance.evaluationDate()).isEqualTo(OffsetDateTime.ofInstant(now.toInstant(), ZoneId.systemDefault()));
+      assertThat(instance.evaluationFailure()).isNull();
+      assertThat(instance.evaluationFailureMessage()).isNull();
+      assertThat(instance.flowNodeInstanceKey()).isEqualTo(migratedFlowNodeInstances.getFirst().flowNodeInstanceKey());
+      assertThat(instance.processInstanceKey()).isEqualTo(migratedProcessInstances.getFirst().processInstanceKey());
+      assertThat(instance.processDefinitionKey()).isEqualTo(migratedProcessInstances.getFirst().processDefinitionKey());
+      assertThat(instance.decisionDefinitionKey()).isEqualTo(migratedDecisions.getFirst().decisionDefinitionKey());
+      assertThat(instance.decisionDefinitionId()).isEqualTo("simpleDecisionId");
+      assertThat(instance.tenantId()).isEqualTo("<default>");
+      assertThat(instance.decisionDefinitionType()).isEqualTo(DecisionInstanceEntity.DecisionDefinitionType.UNSPECIFIED);
+
+      // TODO find out how to get a result http://github.com/camunda/camunda-bpm-platform/issues/5365
+//      assertThat(instance.result()).isEqualTo("B");
+
+      // TODO https://github.com/camunda/camunda-bpm-platform/issues/5364
```

Comment:
C8 ticket is finished, what's missing to do the assertion here?
https://github.com/camunda/camunda/issues/37083

Reply:
Uncommenting the tests and ensuring the mappings re correct. I had not yet implemented the C8 ticket when I opened the PR

Reply:
I was incorrect, there is a little more logic involved that needs to be added here. I will do this with the follow up to speed up progress and for now note as a limitation.